task-list/app/models/task.rb

```diff
@@ -2,8 +2,11 @@ class Task < ActiveRecord::Base
   def self.list
 		sorted_tasks = {completed: [], todo: []}
 		Task.all.each do |task|
-			sorted_tasks[:completed] << task unless task.completed_at.nil?
-			sorted_tasks[:todo] << task if task.completed_at.nil?
+      if task.completed_at.nil?
+        sorted_tasks[:todo] << task if task.completed_at.nil?
```

Comment:
The `if task.completed_at.nil?` part of this line is probably redundant because it is directly within an `if` block that is checking the same thing.